.github/workflows/price-feeder-docker.yml

```diff
@@ -22,7 +22,6 @@ jobs:
         uses: docker/metadata-action@v4
         with:
           images: umee-network/price-feeder
-          tags: type=semver,pattern=v{{version}}
```

Comment:
Why do we need to remove this? I think we should change pattern to `price-feeder/v{{version}}`

Reply:
I think `pattern` is only used to generate tags output , it will not check the release tag 
https://github.com/docker/metadata-action#typesemver

If we add valid semver tag , then only it will generate valid tag output for docker image 

> price-feeder/v{{version}} tag is not a valid semver

If we don't add any tags here , it will take [default tags input]( https://github.com/docker/metadata-action#tags-input), so based on default tags it will generate respective output tag for docker image

Reply:
Can we only keep version as a tagged docker release? I don't think keeping branch or PR makes sense.

Reply:
Default tags are generating below tags for docker image
```json
 "tags": [
      "umee-network/price-feeder:price-feeder-v2.0.2-rc3",
      "umee-network/price-feeder:latest"
    ]
```